Libraries/CMSIS/Device/Maxim/MAX32657/Source/GCC/startup_max32657.S

```diff
@@ -91,6 +154,19 @@ Reset_Handler:
     ldr r0, =__StackTop
     mov sp, r0
 
+#if IS_SECURE_ENVIRONMENT
```

Comment:
`IS_SECURE_ENVIRONMENT` is defined in max32657.h - not sure if this will work in the assembly file

Reply:
Yeah, there are unfinished parts of this PR that I was unsure about. I figure we would need an assembly project flag defined for this when a secure build happens.

Reply:
`PROJ_AFLAGS += IS_SECURE_ENVIRONMENT` added to the secure section of gcc.mk would work well